.storybook/preview.tsx

```diff
@@ -57,30 +57,36 @@ const decorators = [
 
 function CssVariables() {
   const theme = useTheme();
-  const styles = css`
-    ${getMetabaseCssVariables(theme)}
 
-    :root {
-      --mb-default-font-family: "Lato";
+  // This can get expensive so we should memoize it separately
+  const cssVariables = useMemo(() => getMetabaseCssVariables(theme), [theme]);
```

Comment:
Memoizes the CSS variable generation so that we only do it if the theme is updated.